Assets/MRTK/StandardAssets/Shaders/MRTK_Wireframe.shader

```diff
@@ -52,10 +52,14 @@ Shader "Mixed Reality Toolkit/Wireframe"
 
             #include "UnityCG.cginc"
 
-            float4 _BaseColor;
+			CBUFFER_START(UnityPerMaterial)
```

Comment:
nit: looks like some mixed tabs and spaces in some of these files

Reply:
fixed!